src/utils/prompts/blob-delete-prompts.ts

```diff
@@ -0,0 +1,17 @@
+import { log } from '../command-helpers.js'
+
+import { confirmPrompt } from './confirm-prompt.js'
+import { destructiveCommandMessages } from './prompt-messages.js'
+
+export const promptBlobDelete = async (key: string, storeName: string): Promise<void> => {
+  const { overwriteNoticeMessage } = destructiveCommandMessages
+  const { generateWarningMessage, overwriteConfirmationMessage } = destructiveCommandMessages.blobDelete
+
+  const warningMessage = generateWarningMessage(key, storeName)
+
+  log()
+  log(warningMessage)
+  log()
+  log(overwriteNoticeMessage)
+  await confirmPrompt(overwriteConfirmationMessage)
```

Comment:
This reads a lot easier. The way you have it destructing you have to logically read it backwards to understand what it's doing.

Also have a `somethingMessages.somethingElseMessage` is redundant, you don't need the extra `Message` on the end.

```suggestion
  const warningMessage = destructiveCommandMessages.blobDelete.generateWarning(key, storeName)

  log()
  log(warningMessage)
  log()
  log(destructiveCommandMessages.overwriteNotice)
  await confirmPrompt(destructiveCommandMessages.blobDelete.overwriteConfirmation)
```

Reply:
I won't make a comment for each of the `*-prompt.ts` files but it's the same for each